lib/console.rb

```diff
@@ -45,52 +64,49 @@ def default_log_level(env = ENV)
 		
 		# You can change the log level for different classes using CONSOLE_<LEVEL> env vars.
 		#
-		# e.g. `CONSOLE_WARN=Acorn,Banana CONSOLE_DEBUG=Cat` will set the log level for Acorn and Banana to warn and Cat to
-		# debug. This overrides the default log level.
+		# e.g. `CONSOLE_WARN=Acorn,Banana CONSOLE_DEBUG=Cat` will set the log level for the classes Acorn and Banana to `warn` and Cat to `debug`. This overrides the default log level.
 		#
-		# @param logger [Logger] A logger instance to set the logging levels on.
-		# @param env [Hash] Environment to read levels from.
+		# @parameter logger [Logger] A logger instance to set the logging levels on.
+		# @parameter env [Hash] The environment to read levels from.
```

Comment:
Are these directives for the bake world of documentation generation, or a typo?

Reply:
They are picked up by `decode`/`utopia-project` and prefer the unabbreviated forms.